frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -275,11 +276,26 @@ export const funnelLogic = kea<funnelLogicType>({
         people: {
             clearFunnel: () => [],
         },
-        conversionWindowInDays: [
-            14,
+        conversionWindow: [
             {
-                setConversionWindowInDays: (state, { days }) => {
-                    return days >= 1 && days <= 365 ? Math.round(days) : state
+                funnel_window_interval_unit: FunnelConversionWindowTimeUnit.Day,
+                funnel_window_interval: 14,
+            },
+            {
+                setConversionWindow: (
+                    state,
+                    { conversionWindow: { funnel_window_interval_unit, funnel_window_interval } }
+                ) => {
+                    const nextIntervalUnit =
+                        funnel_window_interval_unit ||
+                        state.funnel_window_interval_unit ||
+                        FunnelConversionWindowTimeUnit.Day
+                    const nextInterval = funnel_window_interval || state.funnel_window_interval
+                    return {
+                        ...state,
+                        ...(funnel_window_interval_unit ? { funnel_window_interval_unit: nextIntervalUnit } : {}),
+                        ...(funnel_window_interval ? { funnel_window_interval: nextInterval } : {}),
```

Comment:
Code nit:

If you're checking `funnel_window_interval` exists, doesn't it mean that `nextInterval` being `state.funnel_window_interval` doesn't make sense? Same for  the unit?

Sounds like you can simplify by removing `nextInterval` and `nextIntervalUnit` ?